### PR TITLE
Add script for VoxForge data preparation.

### LINKED ISSUE #9: Need add script to prepare vox dataset.

### PR DESCRIPTION
**Pull request by pkuyym**
Resolves #9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -506,6 +506,8 @@ VoxForge European       |   31.21           |   20.47
 VoxForge Indian         |   56.79           |   28.15
 Baidu Internal Testset  |   47.73           |   8.92
 
+For reproducing benchmark results on VoxForge data, we provide a script to generate VoxForge dialect manifest files. Please go to ```data/voxforge``` and execute ```sh run_data.sh``` to get VoxForge dialect manifest files. Notice that VoxForge data may keep updated and the generated manifest files may have difference from those we evaluated.
```
Comment (kuke) [inline README.md]: ```we provide a script to generate VoxForge dialect manifest files``` -> ```we provide a script to download data and generate VoxForge dialect manifest files```
```may keep updated``` -> ```may keep updating```
```we evaluated``` -> ```we evaluated on```

File: data/voxforge/run_data.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#! /usr/bin/env bash
+
+cd ../.. > /dev/null
+
+# download data, generate manifests
+PYTHONPATH=.:$PYTHONPATH python data/voxforge/voxforge.py \
+--manifest_prefix='data/voxforge/manifest' \
```
Comment (kuke) [inline data/voxforge/run_data.sh]: Now that run the script in the same directory, ```cd ../.. > /dev/null``` and the prefix ```data/voxforge``` won't be necessary.

File: README.md (hunk 1 above)
Comment (pkuyym) [reply README.md]: Done.

File: data/voxforge/run_data.sh (hunk 2 above)
Comment (pkuyym) [reply data/voxforge/run_data.sh]: Done.

**Pull request merged**